app/config.py.example

```diff
@@ -33,3 +33,17 @@ email_conf = ConnectionConfig(
     MAIL_SSL=False,
     USE_CREDENTIALS=True,
 )
+
+# LOGGER
+LOGGER = {
+    "logger": {
```

Comment:
You don't support multiple loggers, so I think it might be better to simplify it:
```
LOG_PATH = ...
LOG_FILENAME = ..
LOG_LEVEL = ...
```

Reply:
I do think it's nicer to receive one parameter in the main module, maybe as a dict still, just without the outer dict "logger"